server/src/main/java/org/elasticsearch/common/io/UTF8StreamWriter.java

```diff
@@ -326,12 +326,4 @@ public void reset() {
         _index = 0;
         _outputStream = null;
     }
-
-    /**
-     * @deprecated Replaced by {@link #setOutput(OutputStream)}
-     */
-    @Deprecated
-    public UTF8StreamWriter setOutputStream(OutputStream out) {
```

Comment:
This is entirely unused now, also deprecated since 6.0. I just piggy-backed it on this PR, can also add it as a separate PR/commit if necessary.